04.ls/ls.rb

```diff
@@ -57,4 +79,92 @@ def puts_table(entry_name_table, max_width)
   end
 end
 
+def create_entry_info_table
+  entry_info_table = Dir.glob('*').map do |entry_name|
+    entry_info = {}
+    stat = File::Stat.new(entry_name)
+    stat_mode = stat.mode.to_s(8)
+    entry_info[:file_mode] = get_file_mode(stat_mode, entry_name)
+    entry_info[:nlink] = stat.nlink.to_s
+    entry_info[:owner_name] = get_owner_name(stat)
```

Comment:
```suggestion
    entry_info[:owner] = get_owner_name(stat)
```

nameはなくしてもいいかもしれません
https://bootcamp.fjord.jp/pages/avoid-too-specific-var-name